reporting_database_generator/elastic_scripts/09_13_elastic_publication_publication.sql

```diff
@@ -0,0 +1,52 @@
+{% set elastic_db = "elastic_production_" + macros.ds_format(macros.ds_add(dag_run.data_interval_end | ds, -1), "%Y-%m-%d", "%Y%m%d") %}
+{% set reporting_db = "pregrant_publications" %}
+
+use `{{elastic_db}}`;
+
+CREATE TABLE IF NOT EXISTS `{{elastic_db}}`.`publication`
```

Comment:
The tables will need to be in their own database or will need to have a prefix. @bethanne-card  Your thoughts?